broker/applications/admin/src/api-controllers/ServiceFabrikAdminController.js

```diff
@@ -31,14 +31,14 @@ const {
 } = require('@sf/common-controllers');
 const bosh = require('@sf/bosh');
 const ScheduleManager = require('@sf/jobs');
-const { maintenanceManager } = require('../../scheduler/maintenance');
+const { maintenanceManager } = require('../../../scheduler/maintenance');
 const { serviceBrokerClient } = require('@sf/broker-client');
 const { apiServerClient } = require('@sf/eventmesh');
-const dbManager = require('../../../data-access-layer/db/DBManager');
-const OobBackupManager = require('../../../data-access-layer/oob-manager/OobBackupManager');
-const DirectorService = require('../../operators/bosh-operator/DirectorService');
+const dbManager = require('../../../../data-access-layer/db/DBManager');
+const OobBackupManager = require('../../../../data-access-layer/oob-manager/OobBackupManager');
+const DirectorService = require('../../../operators/bosh-operator/DirectorService');
```

Comment:
I am fine if you do not take care of these in this PR, but the require statements with path are problems we want to remove.

Reply:
Yes, I am following an incremental approach, I think it could be better if that's done in a separate PR. But either way, I will decide once done with testing of this one.

Reply:
multiple PR is better. Let's focus on having a separate admin process in this one.